docs/installation/installing-with-npm.md

```diff
@@ -1,303 +1,196 @@
+To make GOV.UK Frontend’s images and font work in your project, you can either:
 
-You should [include](#option-1-include-javascript) or [import](#option-2-import-javascript) GOV.UK Frontend JavaScript, and then initialise the script in your application to ensure that all users can use it successfully.
+- route requests to the files in GOV.UK Frontend's folders - recommended
+- copy GOV.UK Frontend's files into your project folder
 
-Note that GOV.UK Frontend does not initialise any scripts by default; all scripts **must** be initialised in order for them to work.
+#### Route requests - recommended
 
-### Option 1: Include JavaScript
+We recommend you route requests for files in `/assets` to the `/node_modules/govuk-frontend/govuk/assets` folder.
 
-Include the `node_modules/govuk-frontend/govuk/all.js` script on your page. You might wish to copy the file into your project or reference it from `node_modules`.
+For example if you're using [express.js](https://expressjs.com/), add the following to your `app.js` file:
 
-To initialise all components, use the `initAll` function.
-
-JavaScript in GOV.UK Frontend requires HTML to be parsed first by the browser before it is initialised. Because of this, make sure you include the script before the closing `</body>` tag.
-Including the script elsewhere will stop components from functioning or displaying correctly.
-
-```html
-    <script src="path-to-assets/govuk-frontend/govuk/all.js"></script>
-    <script>window.GOVUKFrontend.initAll()</script>
-  </body>
-</html>
+```javascript
+var path = require('path');
+app.use('/assets', express.static(path.join(__dirname, '/node_modules/govuk-frontend/govuk/assets')))
 ```
 
-#### Initialise GOV.UK Frontend in only certain sections of a page
-
-By default, the `initAll` function from GOV.UK Frontend initialises all components scoped to an entire page with the `document` object.
+#### Copy files
 
-You can change this by passing the `scope` parameter to the `initAll` function.
+Copy the following folders into your project folder:
 
-For example, if you have a modal dialog box that opens with new markup you could do the following:
-
-```js
-var $modal = document.querySelector('.modal')
-window.GOVUKFrontend.initAll({
-  scope: $modal
-})
-```
+- `/node_modules/govuk-frontend/govuk/assets/images`
+- `/node_modules/govuk-frontend/govuk/assets/fonts`
 
-#### Initialise individual included components
+You should use an automated task or your build pipeline to copy the files, so your project folder stays up to date when we update GOV.UK Frontend.
 
-GOV.UK Frontend components with JavaScript behaviour have the `data-module` attribute set in their markup.
+#### If you use different folders
 
-You can use this attribute to initialise the component manually. This may be useful if you are adding markup to a page after it has loaded.
+If you use different folders for images and the font, you can override GOV.UK Frontend's default paths with your paths.
```

Comment:
I've had a look and I think that with the existing bullet list above this, plus the 2 examples below, users should have enough context to understand this. But we could see how things pan out in research?

Reply:
The examples below don't show any file paths and the two ones above start with `/node_modules` so it might not occur to the user that they're meant to be taking the subdirectory structure from those and reflecting the structure in their project. 

But if you feel confident about it as is then yes keep an eye on it in research.

Reply:
I'm still having doubts about this one, sorry to come back to it. Could we at least say:
> If you use folders for images and the font that are different from the above ones, ....

I don't think it's necessarily clear what it's meant by "different" here. My first thought was "Different from what?" It's only because I know what we talk about here that I was able to figure out it's talking about the folder paths above.

Reply:
@hannalaakso From what we've seen in the pilot version of the research, I think you're probably right, so I'm planning some tweaks based on your thoughts. But I'll see how the rest of the research goes so we can use that to get the final wording right.